config/gateway/gateway-plugin/gateway-plugin.yaml

```diff
@@ -58,6 +58,8 @@ spec:
               value: "6379" 
             - name: AIBRIX_POD_METRIC_REFRESH_INTERVAL_MS
               value: "50"
+            - name: AIBRIX_PREFIX_CACHE_BLOCK_SIZE
```

Comment:
1. Why is there a mismatch between the values in the default and release manifests?

2. Why not keep consistent with vLLM block size?  

using higher block size results in lower hits

Reply:
Current assumption for setting block size is based off gateway's default tokenizer - character based.

1. default is used for local development, hence lower block size because local test strings will have shorter length. For release, increased the block size for actual dataset.

2. vLLM block size is based of tokens where one token can represent multiple chars. For using character based tokenizer, block size of 16 is too low.

Better alternative is to add documentation,
character based tokenizer, default: 16 and release: 64 to 128
tiktoken based tokenizer, default: 4 and release: 16.

Reply:
1. I feel we do not necessarily need to have two set of configuration for dev and release for this setting. keep them same simplify lots of things including support or docs.

2. 128 character size makes sense..

Reply:
Increased the block size to 128 for release.

In vllm if token length is less than block size then it is ignored i.e. hash block is not created. I also moved prefix-cache implementation with same approach. Now if we use default configuration as 128 and some user tests with small string then it will fallback to least-request creating confusion around prefix-cache is not working as expected. 

I am ok to use 128 block size for default and release, with updated documentation to test with message greater than 128 characters. Let me know your final thoughts which option is preferred.

Reply:
For this type of configuration, let's ensure consistency. The only differences between the release and default settings should be related to resource configurations, reflecting their respective resource constraints.